platform-hub-api/db/structure.sql

```diff
@@ -3,7 +3,7 @@
 --
 
 -- Dumped from database version 9.6.1
--- Dumped by pg_dump version 9.6.2
+-- Dumped by pg_dump version 10.1
```

Comment:
uh oh :D whilst it doesn't like like any pg v10 features have impacted this PR, you should probably use the same version of postgres :)

Reply:
whoops, that would have been when I ran `bin/rails db:migrate`. I'll downgrade my local pg version.